ftl/python/builder.py

```diff
@@ -35,9 +35,22 @@ def __init__(self, ctx, args):
             ])
         self._venv_dir = constants.VENV_DIR
         self._wheel_dir = ftl_util.gen_tmp_dir(constants.WHEEL_DIR)
-        self._python_cmd = args.python_cmd.split(" ")
-        self._pip_cmd = args.pip_cmd.split(" ")
-        self._venv_cmd = args.venv_cmd.split(" ")
+
+        if args.python_cmd:
+            self._python_cmd = args.python_cmd.split(" ")
+        else:
+            self._python_cmd = constants.PYTHON_DEFAULT_CMD
```

Comment:
you can make the defaults in args.py constants.****_DEFAULT_CMD instead of having these if statements

Reply:
meant to change this back, thanks. there's a little bit of weirdness going on here since these need to be used as lists, and users pass in a whitespace-separated string, but I can't think of a cleaner way to handle it.